packages/brick_rest_generators/test/rest_model_serdes_generator_test.dart

```diff
@@ -30,6 +32,11 @@ void main() {
         await generateExpectation(
             'unserializable_field_with_generator', unserializable_field_with_generator.output);
       });
+
+      test('constructor_member_field_mismatch', () async {
```

Comment:
it's easier to discover test runs by sync'ing the class name and the test name
```suggestion
      test('RestConstructorMemberFieldMismatch', () async {
```